backend/webapp/src/main/java/org/bonitasoft/web/designer/controller/ConfigurationResource.java

```diff
@@ -52,4 +53,10 @@
         return new ResponseEntity<>(new ConfigurationReport(this.uidVersion, this.modelVersion, bdrUrl), HttpStatus.OK);
     }
 
+    @RequestMapping(value = "/isExperimental", method = RequestMethod.GET)
```

Comment:
Why do you create a separate path for this ? Is there any issue adding this to the ConfigurationReport ?